public/js/services/game.js

```diff
@@ -67,12 +67,35 @@ angular.module('mean.system')
   });
 
   socket.on('gameUpdate', function(data) {
+    console.log('this is game update from public/services/games.js >>>>',data)
```

Comment:
Check through your code for the various places you used console.log, are they really necessary?

Reply:
I apologise, I will do so.